bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx109-en-features.html

```diff
@@ -0,0 +1,118 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% from "macros-protocol.html" import split with context %}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}{{ ftl('whatsnew-page-title') }}{% endblock %}
+{% block page_desc %}{{ ftl('whatsnew-page-description') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('protocol-split') }}
+  {{ css_bundle('firefox_whatsnew') }}
+  {{ css_bundle('firefox_whatsnew_109_en_features') }}
+{% endblock %}
+
+{% set _entrypoint = 'mozilla.org-whatsnew109-features' %}
+{% set _utm_campaign = 'whatsnew109' %}
+
+{% block site_header %}{% endblock %}
+
+{% block wnp_content %}
+  <h1 class="mzp-c-section-heading">Firefox helps you stay private and productive</h1>
+
+  {% call split(
+    block_class='mzp-l-split-center-on-sm-md mzp-l-split-reversed',
+    body_class='mzp-l-split-v-center',
+    media_class='mzp-l-split-h-center',
+    image=resp_img('img/firefox/whatsnew/whatsnew109-en/shield.png',
+      srcset={ 'img/firefox/whatsnew/whatsnew109-en/shield-high-res.png': '2x' },
+      optional_attributes={ 'class': 'mzp-c-split-media-asset'}
+    ),
+  ) %}
+    <h2>See which trackers Firefox blocked for you this week</h2>
+    <p>We use Enhance Tracking Protection to automatically protect your privacy while you browse.</p>
+    <p>
+      <a class="mzp-c-button mzp-t-secondary js-open-about-protections" href="https://support.mozilla.org/kb/enhanced-tracking-protection-firefox-desktop/?utm_source={{ _entrypoint }}&utm_medium=referral&utm_campaign={{ _utm_campaign }}&entrypoint={{ _entrypoint }}"  data-cta-text="View my protections" data-cta-type="button">
```

Comment:
**Suggestion:** A nice suggestion here would be to set a `params` variable on top and use it in the URLs. Similar to what was done in [wnp 108's feature file](https://github.com/mozilla/bedrock/blob/main/bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx108-feature-en.html#L30).

```suggestion
{% set params = 'utm_source=' ~ _entrypoint ~ '&utm_medium=referral&utm_campaign=' ~ _utm_campaign ~ '&entrypoint=' ~ _entrypoint %}


      <a class="mzp-c-button mzp-t-secondary js-open-about-protections" href="https://support.mozilla.org/kb/enhanced-tracking-protection-firefox-desktop/?{{ params }}"  data-cta-text="View my protections" data-cta-type="button">
```